changelog.md

```diff
@@ -1,21 +1,22 @@
-# Changelog
+# Release Notes
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+<br/>
```

Comment:
I'm surprised this line break is needed. Okay if it is, just seems unusual.

In Markdown a line break is usually included if two blank lines are left.